src/Notification/Notification.js

```diff
@@ -0,0 +1,433 @@
+require('moonstone');
+
+/**
+* Contains the declaration for the {@link module:moonstone/Notification~Notification} kind.
+* @module moonstone/Notification
+*/
+
+var
+	kind = require('enyo/kind'),
+	Control = require('enyo/Control'),
+	dispatcher = require('enyo/dispatcher'),
+	EnyoHistory = require('enyo/History'),
+	Popup = require('enyo/Popup'),
+	ShowingTransitionSupport = require('enyo/ShowingTransitionSupport');
+
+var
+	Spotlight = require('spotlight');
+
+var
+	Button = require('../Button'),
+	BodyText = require('../BodyText'),
+	HistorySupport = require('../HistorySupport');
+
+/**
+* {@link module:moonstone/Notification~Notification} is a toast-like minimal popup that comes up
+* from the bottom of the screen.
+*
+* @class Notification
+* @ui
+* @public
+*/
+module.exports = kind(
+	/** @lends module:moonstone/Notification~Notification.prototype */ {
+
+	/**
+	* @private
+	*/
+	name: 'moon.Notification',
+
+	/**
+	* @private
+	*/
+	kind: Popup,
+
+	/**
+	* @private
+	*/
+	mixins: [HistorySupport, ShowingTransitionSupport],
+
+	/**
+	* @private
+	*/
+	showing: false,
+
+	/**
+	* @private
+	*/
+	classes: 'moon moon-neutral enyo-unselectable moon-notification',
+
+	/**
+	* @private
+	*/
+	floating: true,
+
+	/**
+	* @private
+	*/
+	spotlight: 'container',
+
+	/**
+	* @private
+	*/
+	modal: true,
+
+	/**
+	* @private
+	*/
+	spotlightRememberFocus: false,
+
+	/**
+	* @private
+	*/
+	allowDefault: true,
+
+	/**
+	* @private
+	*/
+	handlers: {
+		onRequestScrollIntoView   : '_preventEventBubble',
+		onSpotlightSelect         : 'handleSpotlightSelect'
+	},
+
+	/**
+	* @private
+	*/
+	eventsToCapture: {
+		onSpotlightFocus: 'capturedFocus',
+		onkeydown: 'captureKeyDown'
+	},
+
+	/**
+	* @private
+	* @lends module:moonstone/Popup~Popup.prototype
+	*/
+	published: {
+		/**
+		* When `true`, popups will animate on/off screen.
+		*
+		* @type {Boolean}
+		* @default true
+		* @public
+		*/
+		animate: true,
+
+		/**
+		* The message that will be displayed in the notification's text area.
+		*
+		* @type {String}
+		* @default ''
+		* @public
+		*/
+		content: '',
+
+		/**
+		* When `true`, the notification will animate on/off screen.
+		*
+		* @type {Boolean}
+		* @default true
+		* @public
+		*/
+		showHideDuration: 400,
+
+		/**
+		* If `true`, {@glossary Spotlight} (focus) cannot leave the area of the notification unless
+		* the notification is explicitly closed; if `false`, spotlight may be moved anywhere within
+		* the viewport.
+		*
+		* @type {Boolean}
+		* @default true
+		* @public
+		*/
+		spotlightModal: true,
```

Comment:
As a general note and/or FYI, `spotlightDisabled` has a similar behavior of only being referenced in `Spotlight`.